src/lib.rs

```diff
@@ -123,6 +154,34 @@
 #![deny(missing_docs)]
 #![warn(missing_debug_implementations, rust_2018_idioms)]
 
+
+
+pub use hyper::Method;
```

Comment:
Why move this and the other `use` and `mod` statements from where they were before?